tools/build-fixture.js

```diff
@@ -60,11 +60,11 @@ export const buildDirectory = async fixtureDir => {
 	process.chdir(resolve(fixturePath));
 
 	const parsedOpts = parseScript(script);
-	let output = '';
-	output = await microbundle({
+	let { output } = await microbundle({
```

Comment:
@developit / @marvinhagemeister This is a breaking change if you consider the microbundle js api "official". On the other hand the js api isn't even mentioned in the README - so idk ^.^. What do you think about it?

Reply:
For the future it sure sounds reasonable to make the returning value an `object`. I can imagine it being appended with additional information like metadata, sourcemaps, etc.

Agree that we didn't really advertised the JS-API at all. We haven't even done a proper `1.0.0` release yet, so I'm fine with the breaking change here 👍

Reply:
Awesome! I updated the commit message and its now officially inofficially a breaking change hehe :+1: